website_wow/static/src/xml/snippet_options.xml

```diff
@@ -0,0 +1,537 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License GPL-3 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+
+    <!-- Implement the same interface as Odoo SA in order to be somewhat cross-compatible:
+
+    How to define an animable element in your theme
+    ===============================================
+    <xpath expr="//t[@t-set='o_animate_elements']" position="after">
+        <t t-set="o_animate_elements" t-value="o_animate_elements + ['.myclass']"/>
+    </xpath>
+    ===============================================
+
+    To specifically exclude elements from the possibility of animations, use `o_not-animable` class:
+    ===============================================
+    <section class="o_not-animable" name="My Static Element" />
+    ===============================================
+    -->
+    <template id="snippet_options"
+              name="WOW Options"
+              inherit_id="website.snippet_options">
+
+        <xpath expr="(//div[@data-js])[last()]" position="after">
+
+            <t t-set="o_animate_elements"
+               t-value="['.o_animable', 'section', 'img', '.fa', '[class^=\'col-md\']', '.btn']" />
+            <t t-set="o_wow_selector"
+               t-value="', '.join([s + ':not(.o_not-animable)' for s in o_animate_elements])" />
+
+            <!-- Animations -->
+            <div data-js="o_wow" t-att-data-selector="o_wow_selector">
+                <li class="dropdown-submenu o_wow_options">
+                    <a href="#">
+                        <i class="fa fa-film">Wow!</i>
```

Comment:
Placing the menu text outside the `<i>` tag will keep the styling consistent with the rest of the menu. (same thing applies to the rest of your top-level menu titles)

E.g:
```
<a href="#"><i class="fa fa-film" />Wow!</a>
```